app/web/src/features/dashboard/CommunityBrowser.tsx

```diff
@@ -0,0 +1,166 @@
+import {
+  Divider,
+  Link as MuiLink,
+  List,
+  ListItem,
+  ListItemText,
+} from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import CircularProgress from "components/CircularProgress";
+import {
+  useCommunity,
+  useListSubCommunities,
+} from "features/communities/hooks";
+import { LOAD_MORE } from "features/dashboard/constants";
+import { Community } from "proto/communities_pb";
+import { ReactNode, useState } from "react";
+import { Link } from "react-router-dom";
+import { routeToCommunity } from "routes";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    "& > * + *": {
+      marginInlineStart: theme.spacing(2),
+    },
+  },
+  list: {
+    minWidth: "10rem",
+  },
+  loader: {
+    margin: "auto",
+  },
+  selected: {
+    fontWeight: "bold",
+  },
+}));
+
+export default function CommunityBrowser() {
+  const classes = useStyles();
+  const [selected, setSelected] = useState<Community.AsObject[]>([]);
+
+  const globalCommunityQuery = useCommunity(1);
+
+  //react-query doesn't have useInfiniteQueries
```

Comment:
What do yo mean with react-query does not have useInfinitQueries? :)  https://react-query.tanstack.com/guides/infinite-queries

Reply:
I mean there's no infinite equivalent to `useQueries` which takes an array of query parameters and returns an array of query results. This is sometimes necessary because of the rule of hooks means you can't have a dynamic number of hooks (but you can have a dynamic array in one hook).

Reply:
Sorry, I was indeed to quick with my comment :)